packages/starlight/components/Sidebar.astro

```diff
@@ -28,6 +28,10 @@ const { sidebar, locale } = Astro.props;
     gap: 1rem;
   }
 
+  .sidebar > :global(ul) {
```

Comment:
Thanks for this! I think we want to target the sidebar’s last child for this: a) it means we don’t add a space between the links and the theme/language pickers on mobile and b) it means if we ever support adding some other content to the bottom of the menu, this will also handle that.

```suggestion
  .sidebar > :global(:last-child) {
```